cypress/integration/basic-functionalities/local-tests/one/Search_bar_functionality.spec.js

```diff
@@ -12,8 +12,8 @@ describe('Test for checking the search bar functionality', () => {
 
   context('Test for searching a non existing app', () => {
     it('should not be able to search for a non existing app', () => {
-      // Here we close the start tour pop up (This won't be needed if we are already logged in)
-      //cy.get('.modal-header > :last-child()').click()
+      // Here we close the start tour pop up (This won't be needed if we are already logged in/running the index file)
+      // cy.get('.modal-header > :last-child()').click()
```

Comment:
Do a check if you're logged in or not. Let's not have commented code unless we need to.